phpspy.c

```diff
@@ -738,6 +738,11 @@ static void try_get_php_version(trace_target_t *target) {
             "| grep -Po '(?<=X-Powered-By: PHP/)\\d\\.\\d'",
             pid, pid, pid, pid
         );
+        if (n >= sizeof(version_cmd) - 1) {
+            log_error("version command overflow!");
+            return PHPSPY_ERR;
```

Comment:
Oh woops. I just saw that, while I originally based this commit on some [internal version](https://github.com/Granulate/phpspy/tree/v0.6.0-g3) of ours where `try_get_php_version` has a return value, and in master it has not, so this won't build. Will send a fix promptly

Reply:
Well it does build because we don't have `-Wall`; I did test the build but I ignored warnings :( Will fix anyway

Reply:
Ahh good, I see you've done it in https://github.com/adsr/phpspy/issues/114. Thanks @adsr and sorry for the hassle :)